manage_db.py

```
@@ -0,0 +1,136 @@
+import sys
+from pathlib import Path
+
+# Add project root to path
+sys.path.insert(0, str(Path(__file__).parent))
+
+from src.database.connection import get_db_connection, init_database, check_database_health
+from src.database.seed_data import DatabaseSeeder
+
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s - %(name)s - %(levelname)s - %(message)s'
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)